### ISSUE TITLE
`OTAA_Unconfirmed_With_Cloud_To_Device_Mac_Command_Returns_Downstream_Message` fails sporadically

### ISSUE THREAD

**Issue by bastbu** [Azure/iotedge-lorawan-starterkit#745]
## Expected Behavior

Integration test `LoRaWan.Tests.Integration.CloudToDeviceMessageTests.OTAA_Unconfirmed_With_Cloud_To_Device_Mac_Command_Returns_Downstream_Message` should not fail.

## Current Behavior

Integration test fails (sometimes) with error message:

```
System.Net.NetworkInformation.NetworkInformationException : An error was encountered while querying information from the operating system.
---- System.AggregateException : One or more errors occurred. (Could not find a part of the path '/sys/class/net/vethfc3b88e/mtu'.)
-------- System.IO.DirectoryNotFoundException : Could not find a part of the path '/sys/class/net/vethfc3b88e/mtu'.
```

Stack trace:

```
System.Net.NetworkInformation.LinuxNetworkInterface.GetLinuxNetworkInterfaces()
LoRaWan.Tests.Common.Utility.GetMacAddress()
LoRaWan.Tests.Common.MessageProcessorTestBase..ctor()
LoRaWan.Tests.Integration.CloudToDeviceMessageTests..ctor()
System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean skipCheckThis, Boolean fillCache, Boolean wrapExceptions)
Interop.ThrowExceptionForIoErrno(ErrorInfo errorInfo, String path, Boolean isDirectory, Func`2 errorRewriter)
Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String path, OpenFlags flags, Int32 mode)
System.IO.FileStream.OpenHandle(FileMode mode, FileShare share, FileOptions options)
System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options)
System.IO.StreamReader.ValidateArgsAndOpenPath(String path, Encoding encoding, Int32 bufferSize)
System.IO.StreamReader..ctor(String path, Encoding encoding, Boolean detectEncodingFromByteOrderMarks)
System.IO.File.InternalReadAllText(String path, Encoding encoding)
System.IO.File.ReadAllText(String path)
System.Net.NetworkInformation.StringParsingHelpers.ParseRawIntFile(String filePath)
System.Net.NetworkInformation.LinuxIPv4InterfaceProperties.GetMtu()
System.Net.NetworkInformation.LinuxIPv4InterfaceProperties..ctor(LinuxNetworkInterface linuxNetworkInterface)
System.Net.NetworkInformation.LinuxIPInterfaceProperties..ctor(LinuxNetworkInterface lni)
System.Net.NetworkInformation.LinuxNetworkInterface..ctor(String name, Int32 index)
System.Net.NetworkInformation.LinuxNetworkInterface.GetOrCreate(Dictionary`2 interfaces, String name, Int32 index)
System.Net.NetworkInformation.LinuxNetworkInterface.<>c__DisplayClass5_0.<GetLinuxNetworkInterfaces>b__2(String name, LinkLayerAddressInfo* llAddr)
```

## Steps to Reproduce

- Run: https://github.com/Azure/iotedge-lorawan-starterkit/actions/runs/1435989612
- Test results: https://github.com/Azure/iotedge-lorawan-starterkit/runs/4142562853#r0s5

**Comment by bastbu** [comment]
Might be related to https://github.com/dotnet/runtime/issues/50293 and be fixed when upgrading to .NET 6 runtime.

**Comment by roel4ez** [comment]
this will auto resolve as part of #817

**Comment by bastbu** [comment]
Closed by #817
